go.mod

```diff
@@ -65,11 +65,6 @@ replace (
 	google.golang.org/grpc => google.golang.org/grpc v1.27.0
 	google.golang.org/grpc/examples => google.golang.org/grpc/examples v0.0.0-20200709232328-d8193ee9cc3e
 
-	// pinned for delegated authentication watch request bug fix.
-	k8s.io/apimachinery => k8s.io/apimachinery v0.21.0-beta.1.0.20210308143346-a13af1068ef1
-	k8s.io/apiserver => k8s.io/apiserver v0.0.0-20210409112051-49d90ce0ad13
-	k8s.io/component-base => k8s.io/component-base v0.0.0-20210105235135-9c158118ed58
```

Comment:
I double-checked the root vendor had the changes listed in https://github.com/kubernetes/apiserver/commit/49d90ce0ad135a6bfd06deb5cf53e47b7945ba1c.